src/components/toast/toast.pw.tsx

```diff
@@ -0,0 +1,334 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+
+import { ToastProps } from ".";
+import {
+  ToastComponent,
+  ToastWhenOtherModalRenders,
+  ToastAllAlign,
```

Comment:
ToastComponent and ToastAllAlign can probably be removed from toast-test.stories.tsx